ownership_type/ownertype_script.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+
+import progressbar
+from typing import Iterable
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.emitter.rest_emitter import DataHubRestEmitter
+from datahub.ingestion.graph.client import DataHubGraph
+from datahub.ingestion.graph.config import DatahubClientConfig
+from datahub.metadata._schema_classes import (
+    OwnershipClass,
+    OwnerClass,
+    OwnershipSourceClass,
+    OwnershipTypeClass,
+)
+
+# setup below
+TOKEN = "xxx"
+HOSTNAME = "https://xxx.acryl.io/gms"
+
+graph = DataHubGraph(
+    config=DatahubClientConfig(server=HOSTNAME, token=TOKEN if TOKEN else None)
+)
+emitter = DataHubRestEmitter(gms_server=f"{HOSTNAME}", token=TOKEN if TOKEN else None)
+
+scroll_container_query = """
+query scrollAcrossEntities($input: ScrollAcrossEntitiesInput!) {
+  scrollAcrossEntities(input: $input) {
+    nextScrollId
+    count
+    total
+    searchResults {
+      entity {
+        urn
+      }
+    }
+  }
+}
+"""
+
+
+search_query = """
+query getContainer($urn: String!) {
+  container(urn: $urn) {
+    entities(input: {start: 0, count: 10000}) {
+      total
+      searchResults {
+        entity {
+          urn
+          ... on Dataset {
+            ownership {
+              owners {
+                owner {
+                  __typename
+                  ... on CorpUser {
+                    urn
+                  }
+                  ... on CorpGroup {
+                    urn
+                  }
+                }
+                ownershipType {
+                  urn
+                  type
+                  info {
+                    name
+                  }
+                }
+                source {
+                  type
+                  url
+                }
+              }
+              lastModified {
+                time
+                actor
+              }
+            }
+          }
+        }
+      }
+      __typename
+    }
+  }
+}
+"""
+
+
+dataset_done = set()
+
+
+def get_container_urns() -> Iterable[str]:
```

Comment:
This seems like a duplicate of `dataHubGraph.get_urns_by_filter` command which is part of our SDK